### ISSUE TITLE
Will not work with AudioKit v3

### ISSUE THREAD

**Issue by csann** [elslooo/guitar-tuner#1]
Please upgrade this to v3!

Please, please, please

**Comment by addy28** [comment]
@csann  is it working with previous versions ??

**Comment by csann** [comment]
Sorry. I don’t have any previous versions of AudioKit to test it with. AudioKit made many changes from v2 to v3.

**Comment by denisb411** [comment]
Yes, it is still working with the newest swift. You just have to keep using AudioKit 2.2, and of course, make some code adaptations (of which 90% xcode do it for you).

**Comment by elslooo** [comment]
For what it's worth, you can get AudioKit 2.x by default by running `pod install`. The problem with AudioKit 3.x is that they replaced the algorithm that is relevant to my blog posts with something completely different (autocorrelation if I remember correctly).

**Comment by Sm038229** [comment]
By running 'pod install' it installs AudioKit 3.5.3 and gives an error as 'Module compiled with Swift 3.0.2 cannot be imported in Swift 3.1'. So please give a solution for it in XCode 8.3.2.

**Comment by denisb411** [comment]
@Sm038229 you have to manually modify the version you want to install. Follow the instructions: https://elsl.ooo/2016/02/12/digital-tuner-ios-swift.html

**Comment by doriansgithub** [comment]
what would be a rough estimate of the time necessary to update this to AK4?

**Comment by doriansgithub** [comment]
Loading AudioKit 2.3 unto our project, while using the xcworkspace, produces two errors:

error: Invalid bitcode signature
clang: error: linker command failed with exit code 1 (use -v to see invocation)
